test/support/test_config.h

```diff
@@ -192,7 +192,8 @@
 
 // oneAPI DPC++ compiler 2022.2 an below show an internal compiler error during the backend code generation of
 // minmax_element.pass.cpp
-#define _PSTL_ICPX_TEST_MINMAX_ELEMENT_BROKEN                                                                         \
-    (TEST_DPCPP_BACKEND_PRESENT && defined(__INTEL_LLVM_COMPILER) && __INTEL_LLVM_COMPILER < 20220300)
+#if TEST_DPCPP_BACKEND_PRESENT && defined(__INTEL_LLVM_COMPILER) && __INTEL_LLVM_COMPILER < 20220300
+#   define _PSTL_ICPX_TEST_MINMAX_ELEMENT_BROKEN 1

```

Comment:
It is not quite correct, due to _PSTL_ICPX_TEST_MINMAX_ELEMENT_BROKEN might be undefined when condition on line 195 is false

Reply:
One option would be to just change `defined(__INTEL_LLVM_COMPILER)` to `__INTEL_LLVM_COMPILER > 0`, and have no other changes. 

As it is written, the check is if `!defined(_PSTL_ICPX_TEST_MINMAX_ELEMENT_BROKEN)` so I think this does work, but its a little odd to set to `1` then check if defined rather than if `true`